addon/components/o-s-s/power-select.hbs

```diff
@@ -0,0 +1,31 @@
+<div class="upf-power-select fx-col" ...attributes {{did-insert (fn this.ensureBlockPresence 'selected-item' (has-block 'selected-item'))}}
+  {{did-insert (fn this.ensureBlockPresence 'option-item' (has-block 'option-item'))}}>
```

Comment:
Can't you send both "has-block booleans check" at the same time ? 🤔 
`{{did-insert (fn this.ensureBlockPresence 'selected-item' (has-block 'selected-item') (has-block 'option-item'))}}`
And have the `ensureBlockPresence` method take two params ?

Reply:
Yes I can try